stripe2qbo/api/routers/transaction_router.py

```diff
@@ -0,0 +1,47 @@
+from typing import Annotated, List
+
+from sqlalchemy.orm import Session
+from fastapi import APIRouter, Depends, HTTPException
+
+from stripe2qbo.api.dependencies import get_current_user, get_db
+from stripe2qbo.db.models import User, TransactionSync as TransactionSyncORM
+
+from stripe2qbo.db.schemas import TransactionSync
+
+router = APIRouter(
+    prefix="/transaction",
+    tags=["transaction"],
+)
+
+
+@router.get("/")
+async def get_all_transactions(
+    user: Annotated[User, Depends(get_current_user)],
+    db: Annotated[Session, Depends(get_db)],
+) -> List[TransactionSync]:
+    transactions = (
+        db.query(TransactionSyncORM).filter(TransactionSyncORM.user_id == user.id).all()
+    )
+    if transactions is None:
+        raise HTTPException(status_code=404, detail="No transactions found")
```

Comment:
No need to raise an exception here. I think we can just return an empty list 

Actually, that is what I would expect from `.all()` can `transactions` even be None?